js/data/move_data.js

```diff
@@ -730,19 +867,18 @@ var MOVES_ADV = $.extend(true, {}, MOVES_GSC, {
         makesContact: true,
         isPunch: true
     },
-    'Spark': {
-        bp: 65,
-        type: 'Electric',
-        category: 'Physical',
-        makesContact: true,
-        hasSecondaryEffect: true
-    },
     'Superpower': {
         bp: 120,
         type: 'Fighting',
         category: 'Physical',
         makesContact: true
     },
+    'Uproar': {
+        bp: 50,
+        type: 'Normal',
+        category: 'Special',
+        isMultiHit: true
```

Comment:
This isn't accurate; Uproar is like Outrage and friends, not stuff like Rock Blast.